safe_autonomy_simulation/sims/spacecraft/rotational_model.py

```diff
@@ -116,6 +116,8 @@ def __init__(
         material: mat.Material = defaults.CWH_MATERIAL,
         parent: typing.Union[e.PhysicalEntity, None] = None,
         children: list[e.PhysicalEntity] = [],
+        control_min: typing.Union[float, np.ndarray, None] = None,
+        control_max: typing.Union[float, np.ndarray, None] = None,
```

Comment:
Fine as is being a direct override of anything computed using the angular limits. However this behavior should be noted in a docstring entry for each of these new parameters. Something indicating that if this value is None, defaults computed using angular limits will be used.

Reply:
Please check that doc strings have been updated satisfactorily.